Jenkinsfile

```diff
@@ -38,5 +38,4 @@ node('build-slave') {
         currentBuild.result = "FAILURE"
         throw err
     }
```

Comment:
Still seeing changes in Jenkin file can you please copy as it is.
We don't need to change this file at all